### DISCUSSION TITLE
Just add a new row?

### DISCUSSION THREAD

**Discussion by TomAguero** [dfinke/ImportExcel#1484] [Q&A]
I feel like I'm missing something that should be extremely simple, I just want to add data to a new row at the bottom of 1 worksheet. Everything I've tried so far either does nothing or creates a new worksheet. I probably need to provide more info to help, but I'm pretty frustrated and have spent enough time on this for now.

**Comment by geoced** [comment]
Data usually come from PowerShell objects being piped to Export-Excel. 

So if you need a new row, just add a new object to the array you export to Excel.

If you need an empty row, create a object with $null properties.

Otherwise, you can manipulate Excell cells directly like this:

`$xlPackage.SheetName.Cells["A42"].Value = "your data"`
